integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -1724,25 +1759,61 @@ func (v *EthereumKeeperRegistrar) EncodeRegisterRequest(
 		}
 		return req, nil
 	} else if v.registrar21 != nil {
-		req, err := registrarABI.Pack(
-			"register",
-			name,
-			email,
-			common.HexToAddress(upkeepAddr),
-			gasLimit,
-			common.HexToAddress(adminAddr),
-			uint8(0), // trigger type
-			checkData,
-			[]byte{}, // triggerConfig
-			[]byte{}, // offchainConfig
-			amount,
-			common.HexToAddress(senderAddr),
-		)
+		if isLogTrigger {
+			logTriggerConfigType := goabi.MustNewType("tuple(address contractAddress, uint8 filterSelector, bytes32 topic0, bytes32 topic1, bytes32 topic2, bytes32 topic3)")
```

Comment:
can we use the automationUtils contract to pack this struct instead? It adds much better type safety :)